resources/web/wwi/test.html

```diff
@@ -1,136 +0,0 @@
-<!DOCTYPE html>
-
-<style>
```

Comment:
perhaps it isn't a good idea, but can't we have a default html included in the repository (but which we don't include in the distro)?
Otherwise every time we want to debug/test something locally we need to create one